GoogleForCreators/web-stories-wp
# Editor and output: Custom page advancement

## Context

This adds a new custom per-page page advancement panel.

## User-facing changes

| Advancement setting | Without override | With override |
|-|-|-|
| Manual | <img width="357" alt="Screen Shot 2022-09-14 at 19 00 12" src="https://user-images.githubusercontent.com/637548/190264560-f296fea8-57b8-43dc-8d27-8b098eb0e50c.png"> | <img width="357" alt="Screen Shot 2022-09-14 at 18 58 37" src="https://user-images.githubusercontent.com/637548/190264574-21861ae5-8399-44f7-b635-2df0e16e15ff.png"> |
| Auto | <img width="357" alt="Screen Shot 2022-09-14 at 18 58 23" src="https://user-images.githubusercontent.com/637548/190264600-b31fb736-b306-407b-acd4-79a7c18ea769.png"> | <img width="357" alt="Screen Shot 2022-09-14 at 18 58 33" src="https://user-images.githubusercontent.com/637548/190264583-c65a70c4-108d-4272-a1b8-65f573bd9425.png"> |

## Testing Instructions

This PR can be tested by following these steps:

1. Enable the experiment
2. Set any document-wide page advancement setting in the document panel
3. Then for any individual page, override the global page advancement setting locally (or not)
4. Observe that the preview story will correctly used either the local page setting or the global setting if not overridden.

## Checklist




---

Fixes #12210
